async-nats/src/lib.rs

```diff
@@ -578,6 +592,10 @@ impl Connector {
                                 }
                             }
 
+                            Some(ServerOp::Error(error_message)) => {
+                                println!("error from server: {:?}", error_message);
```

Comment:
Note to us: let's follow up with either log or tracing crate and debug dump everything.